### PR TITLE
Created API service, which can get movies data from OMDb and stores i…

### PR DESCRIPTION
**Pull request by ASavka**
…t in RAM

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -2,37 +2,142 @@ import express, { Request, Response, Application, NextFunction } from 'express';
 import { config } from './config';
 import logger from './logger';
 import { v4 as uuid } from 'uuid';
+import axios from 'axios';
+import * as _ from 'underscore';
```
Comment (rfist) [inline src/server.ts:6]: You should add `underscore` library to package.json

File: src/server.ts [hunk 2]
```diff
@@ -2,37 +2,142 @@ import express, { Request, Response, Application, NextFunction } from 'express';
...
 
+const url = 'http://www.omdbapi.com?apikey=16d8c738&t=';
```
Comment (rfist) [inline src/server.ts:8]: Just `url` is not the best constant name, try to use something more distinct, like `OMDB_BASE_URL`

File: src/server.ts [hunk 3]
```diff
@@ -2,37 +2,142 @@ import express, { Request, Response, Application, NextFunction } from 'express';
...
+  res: Response,
+  next: NextFunction
+) => {
+  logger.error(err);
+  res.status(500).send(err.message);
+
+  next();
+};
+
+const verifyMoviePresent = (id: string) => {
```
Comment (rfist) [inline src/server.ts:40]: Can't see any usage of this function, do we need actually need it?

**Review by rfist — APPROVED** [review]
Good job!

File: src/server.ts (hunk 3 above)
Comment (ASavka) [reply src/server.ts:40]: No. Removed

File: src/server.ts (hunk 2 above)
Comment (ASavka) [reply src/server.ts:8]: Done

File: src/server.ts (hunk 1 above)
Comment (ASavka) [reply src/server.ts:6]: Added (forgot to install with --save-dev)

**Pull request merged**